rust/headless-client/src/main.rs

```diff
@@ -201,8 +201,14 @@ fn main() -> Result<()> {
     };
     Telemetry::set_firezone_id(firezone_id.clone());
 
+    analytics::identify(
+        firezone_id.clone(),
```

Comment:
Could you update this to `sha256` the id before sending? That way it will match prod data. Hit a snag associating these between Sentry and DB because they are sha256 before hitting the control plane request.

Reply:
I'll migrate the existing Firezone IDs in a different PR instead.